### PR TITLE
Appgwfileshare

### PR DESCRIPTION
**Pull request by migel0**

### CODE REVIEW

File: azuredeploy-large-ha.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
             "metadata": {
                 "description": "The base URI where artifacts required by this template are located. When the template is deployed using the accompanying scripts, a private location in the subscription will be used and this value will be automatically generated."
             },
-            "defaultValue": "https://raw.githubusercontent.com/Azure/Moodle/master/"
+            "defaultValue": "https://raw.githubusercontent.com/migel0/Moodle/edit/appgwfileshare/"
```
Comment (rgardler) [inline azuredeploy-large-ha.json:10]: This moves the authoratative repo to your own person repo. This cannot be changed. Furthermore the URL is a 400 and thus will not work.

File: azuredeploy-large-ha.json [hunk 2]
```diff
@@ -39,8 +39,11 @@
                     "mysqlPgresVcores":             { "value": 16 },
                     "mysqlPgresStgSizeGB":          { "value": 512 },
                     "fileServerType":               { "value": "azurefiles" },
-                    "fileServerDiskSize":           { "value": 1024 },
-                    "storageAccountType":           { "value": "Premium_LRS" }
+                    "fileServerDiskSize":           { "value": 512 },
+                    "storageAccountType":           { "value": "Premium_LRS" },
+                    "httpsTermination":             { "value": "AppGw" },
+                    "sslCertKeyVaultURL":           { "value": "https://kvmoodlepoc.vault.azure.net/secrets/SECRETCERT2/439f06bc3e884df88c9343af851fe154" },
```
Comment (rgardler) [inline azuredeploy-large-ha.json:45]: This is a cert unique to your install and should not be committed to Git.

File: nested/appgw.json [hunk 3]
```diff
@@ -83,13 +83,12 @@
...
...4Bt1OcIbsTPjasE6ULo1dtbkJGxhVXaMFBo7Zgo8cEDdHbhudOg1KpGlM9UMlCI7EhPdeX+ybo2D18RtsV9pmFveSwq1mRxuFOH7j1mPzxvvc+F4c8/qCTx9B2nb+x+pQMGrJKSh7bnqE2DrBH7PmHqceD60CqR1W5qsPc1g4GAhtR3tH867Og/O4+9rw9Wc/+IKgN3YiOP0nMBIjqdnkFdOxijUsLIzl0C/rjV0q0WzjPmDU4h0sRJKaPFblezLiyE0xYo6aJFt1eBLVx6tkESiBf4DnVJiOWazm4h/B2v56HC5v+uJYmzel3w/Xrtwni7L7D/C3QlovmAK8lHqn5arfNkn2aEC2Xp0TwXqu59NhFL7GigYuDnWVp3MgbcoBs8nv1GGmIT3PjSZl8YqiMTNm86zr4nrAsbaZnJIss7mXVMNNiwFFMr3FaaLwbayg9FMbYeAjtlvS6hoyeQTzrtkO9Wvpy8zRlosId8pCGDfPvISWFo61jdserd4Ok+OuasjRLj6FdXlVzgb17Kg8EP02NI+XxijpNiAgELAIWr3K4aEKV3bjcwzEjT70zFqt3A6QFvSq5KXU4Z7J8YU+z5MGkQmaSk/ES4CToqsw1CYxo3/DNdnuHq2YtkIqMuO3knO+yyTKTnRWjaq1J4A74hGfF8ebeVij8HvLaoxWva0YMPwcQ8MG2o7ahZuPr7BUhRuwwp0dGza+Hpj7nm3d28qekOAKAIvIbNaoGFT2KZvIvKaZJlV6aXyO6v3T6HSGB9V9b5kl8XEc/r+Vz0O9FmS7vU785Lr1H8B1Tck84LL5qvH5j5xtaNuuX0Re3QWh2AZwRzYBGBB8ESlm36ZjEk7wCEMPZvjzEBj3WziyIQE9iehJt6CNjP0Gog7Us83DJT1/6LvaZtJHtY5TLk9ZbCmcH1OwhvwoJ8TkeTG9RTOt0BUyWvLkE5CrJJu6ONXm8JdrkkTbcFN9NlE/zO4lK6vuP2BXcWGFGt3IKe+3ZhK91AC1xKMX6c9g/kWILLi8RFnkYZFramhhW0sdp9YB5JlDEGADC1uqxsPDAGgyE1age2NFJd0kYxJTAjBgkqhkiG9w0BCRUxFgQUz7QKW0MUN0DggVujbODK48N2vtcwMTAhMAkGBSsOAwIaBQAEFGeus2X6eMXVPu6r+nbET99oJ8hVBAh+ZDsUTG6CdwICCAA="
```
Comment (rgardler) [inline nested/appgw.json:89]: Should be unique per deployment.

File: nested/network.json [hunk 4]
```diff
@@ -288,12 +288,7 @@
...
...4Bt1OcIbsTPjasE6ULo1dtbkJGxhVXaMFBo7Zgo8cEDdHbhudOg1KpGlM9UMlCI7EhPdeX+ybo2D18RtsV9pmFveSwq1mRxuFOH7j1mPzxvvc+F4c8/qCTx9B2nb+x+pQMGrJKSh7bnqE2DrBH7PmHqceD60CqR1W5qsPc1g4GAhtR3tH867Og/O4+9rw9Wc/+IKgN3YiOP0nMBIjqdnkFdOxijUsLIzl0C/rjV0q0WzjPmDU4h0sRJKaPFblezLiyE0xYo6aJFt1eBLVx6tkESiBf4DnVJiOWazm4h/B2v56HC5v+uJYmzel3w/Xrtwni7L7D/C3QlovmAK8lHqn5arfNkn2aEC2Xp0TwXqu59NhFL7GigYuDnWVp3MgbcoBs8nv1GGmIT3PjSZl8YqiMTNm86zr4nrAsbaZnJIss7mXVMNNiwFFMr3FaaLwbayg9FMbYeAjtlvS6hoyeQTzrtkO9Wvpy8zRlosId8pCGDfPvISWFo61jdserd4Ok+OuasjRLj6FdXlVzgb17Kg8EP02NI+XxijpNiAgELAIWr3K4aEKV3bjcwzEjT70zFqt3A6QFvSq5KXU4Z7J8YU+z5MGkQmaSk/ES4CToqsw1CYxo3/DNdnuHq2YtkIqMuO3knO+yyTKTnRWjaq1J4A74hGfF8ebeVij8HvLaoxWva0YMPwcQ8MG2o7ahZuPr7BUhRuwwp0dGza+Hpj7nm3d28qekOAKAIvIbNaoGFT2KZvIvKaZJlV6aXyO6v3T6HSGB9V9b5kl8XEc/r+Vz0O9FmS7vU785Lr1H8B1Tck84LL5qvH5j5xtaNuuX0Re3QWh2AZwRzYBGBB8ESlm36ZjEk7wCEMPZvjzEBj3WziyIQE9iehJt6CNjP0Gog7Us83DJT1/6LvaZtJHtY5TLk9ZbCmcH1OwhvwoJ8TkeTG9RTOt0BUyWvLkE5CrJJu6ONXm8JdrkkTbcFN9NlE/zO4lK6vuP2BXcWGFGt3IKe+3ZhK91AC1xKMX6c9g/kWILLi8RFnkYZFramhhW0sdp9YB5JlDEGADC1uqxsPDAGgyE1age2NFJd0kYxJTAjBgkqhkiG9w0BCRUxFgQUz7QKW0MUN0DggVujbODK48N2vtcwMTAhMAkGBSsOAwIaBQAEFGeus2X6eMXVPu6r+nbET99oJ8hVBAh+ZDsUTG6CdwICCAA="
```
Comment (rgardler) [inline nested/network.json:291]: Should be unique per deployment

**Comment by migel0** [comment]
Sorry, my fault, i was doing a POC ( no real data on this amount of secrets, etc)

**Comment by migel0** [comment]
Opened by error

**Pull request closed without merging**